shoop/testing/factories.py

```diff
@@ -411,8 +412,11 @@ def create_product(sku, shop=None, supplier=None):
     product.save()
     if shop:
         sp = ShopProduct.objects.create(product=product, shop=shop)
+        if default_price:
```

Comment:
This should probably be `if default_price is not None:`, so users could set zero here